charts/azure-arc-sample/templates/deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Release.Name}}
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{.Release.Name}}
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  minReadySeconds: 5 
+  template:
+    metadata:
+      labels:
+        app: {{.Release.Name}}
+    spec:
+      nodeSelector:
+        kubernetes.io/arch: amd64
+        kubernetes.io/os: linux
+      containers:
+      - name: {{.Release.Name}}
+        image: akashkeshari3010/arc-conformance-test:jsappv1
```

Comment:
Now that we've added pipeline YAML in this PR, we should change this as well. Just push the image once manually to ACR before we merge this PR. When we merge this PR to master branch, configurations already created will try to deploy this chart. Image pull shouldn't fail then